plugins/redis/redis.cpp

```diff
@@ -167,7 +174,10 @@ protected :
     void assertConnection(const char * _msg);
     void assertConnectionWithCmdMsg(const char * cmd, const char * key = NULL);
     void fail(const char * cmd, const char * errmsg, const char * key = NULL);
-    void * redisCommand(redisContext * context, const char * format, ...);
+    void * redisCommand(const char * format, ...);
+    void strto(const char * str, double & ret, const char * key);
```

Comment:
style: Generally convention is to either have output parameters first in the list or last in the list.  (I prefer them 1st, google style is for them to be last.)